build.gradle

```diff
@@ -3,9 +3,8 @@ plugins {
     id 'application'
     id 'org.openjfx.javafxplugin' version '0.1.0'
     id 'idea'
-    id 'org.springframework.boot' version '3.3.2'
+    id 'org.springframework.boot' version '2.7.5'
     id 'io.spring.dependency-management' version '1.1.6'
-    id 'com.adarshr.test-logger' version '4.0.0'
```

Comment:
was causing errors like `Type org.gradle.api.tasks.testing.TestFailure not present
`

Reply:
also this is only for aesthetic reasons afaict so it's not dangerous to delete